<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>155</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Serial No:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Item Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Make:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Price:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Purchase Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Warranty Valid Till:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>User:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>277</width>
      <height>400</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="serialNoBox">
       <property name="maxLength">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="itemTypeBox">
       <item>
        <property name="text">
         <string>Select</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="descriptionBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Select</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="makeBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Select</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modelBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Select</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="priceBox">
       <property name="maxLength">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="purchaseYYYY">
         <item>
          <property name="text">
           <string>YYYY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2005</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2006</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2007</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2008</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2009</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2010</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2011</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2012</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2013</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2014</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2015</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2016</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2017</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2018</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2019</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2020</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2021</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="purchaseMM">
         <item>
          <property name="text">
           <string>MM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>00</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>02</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>04</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>08</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>09</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="purchaseDD">
         <item>
          <property name="text">
           <string>DD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>00</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>02</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>04</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>08</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>09</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>31</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QComboBox" name="validYYYY">
         <item>
          <property name="text">
           <string>YYYY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2005</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2006</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2007</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2008</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2009</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2010</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2011</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2012</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2013</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2014</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2015</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2016</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2017</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2018</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2019</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2020</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2021</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="validMM">
         <item>
          <property name="text">
           <string>MM</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>00</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>02</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>04</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>08</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>09</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="validDD">
         <item>
          <property name="text">
           <string>DD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>00</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>02</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>04</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>08</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>09</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>31</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="locationBox">
       <item>
        <property name="text">
         <string>Select</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="userBox">
       <item>
        <property name="text">
         <string>Select</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>401</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>33</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
